src/styles/index.ts

```diff
@@ -4,16 +4,19 @@ import { CSSBreakpoints } from 'shared/constants';
 
 import theme from './theme';
 
+export enum FlashMessageType {
+  danger = 'danger',
+  success = 'success',
+  warning = 'warning',
+  info = 'info',
```

Comment:
Enum members are [usually](https://www.typescriptlang.org/docs/handbook/enums.html) PascalCased

```suggestion
  Danger = 'danger',
  Success = 'success',
  Warning = 'warning',
  Info = 'info',
```